src/geometry/quaternion.rs

```diff
@@ -537,6 +537,29 @@ impl<N: Real> Quaternion<N> {
         Self::from_imag(self.imag())
     }
 
+    /// Calculate left quaternionic division.
+    ///
+    /// # Example
+    /// ```
+    /// # #[macro_use] extern crate approx;
+    /// # use nalgebra::Quaternion;
+    /// let a = Quaternion::new(0.0, 1.0, 2.0, 3.0);
+    /// let b = Quaternion::new(0.0, 5.0, 2.0, 1.0);
+    /// let result = a.left_div(&b).unwrap();
+    /// let expected = Quaternion::new(0.4, 0.13333333333333336, -0.4666666666666667, 0.26666666666666666);
+    /// assert_relative_eq!(expected, result, epsilon = 1.0e-7);
+    /// ```
+    #[inline]
+    pub fn left_div(&self, other: &Self) -> Option<Self> {
+        other.inv().map(|inv| self * inv)
```

Comment:
Shouldn't this be `self.inv().map(|inv| inv * other)` instead? I'm guessing "left division" refers to: `a \ b = a⁻¹b` where `a` is `self` and `b` is `other`.